awx/main/utils/common.py

```diff
@@ -739,16 +758,8 @@ def get_mem_capacity(raw_mb=None):
     elif settings_absmem is not None:
         return 0, int(settings_absmem)
 
-    if env_forkmem:
-        forkmem = int(env_forkmem)
-    elif settings_forkmem:
-        forkmem = int(settings_forkmem)
-    else:
-        forkmem = 100
-
-    if raw_mb is None:
-        raw_mb = psutil.virtual_memory().total
-    return (raw_mb, max(1, ((raw_mb // 1024 // 1024) - 2048) // forkmem))
+    mem = psutil.virtual_memory().total
+    return (mem, get_mem_effective_capacity(mem))
```

Comment:
@shanemcd I believe this still respects the SYSTEM_TASK_FORKS_MEM / SYSTEM_TASK_FORKS_CPU settings the same way it did before.

This will not respect SYSTEM_TASK_ABS_MEM / SYSTEM_TASK_ABS_CPU. My thinking is that it doesn't make sense for execution nodes. I could be wrong, so I wanted to ping you.

I still can't find any example of these being used for the life of me. Can you link me to anything?